PWGCF/Femto3D/DataModel/singletrackselector.h

```diff
@@ -0,0 +1,201 @@
+DECLARE_SOA_DYNAMIC_COLUMN(TOFNSigmaDe, tofNSigmaDe,
+                           [](nsigma::binning::binned_t nsigma_binned) -> float { return singletrackselector::unPack<nsigma::binning>(nsigma_binned); });
+DECLARE_SOA_DYNAMIC_COLUMN(TPCNSigmaDe, tpcNSigmaDe,
+                           [](nsigma::binning::binned_t nsigma_binned) -> float { return singletrackselector::unPack<nsigma::binning>(nsigma_binned); });
+
+} // namespace singletrackselector
+
+DECLARE_SOA_TABLE_FULL(SingleTrackSel, "SelTracks", "AOD", "STSEL", // Table of the variables for single track selection.
+                       o2::soa::Index<>,
+                       singletrackselector::SingleCollSelId,
+                       singletrackselector::P,
+                       singletrackselector::DcaXY,
+                       singletrackselector::DcaZ,
+                       singletrackselector::TPCInnerParam,
+                       singletrackselector::TPCSignal,
+                       singletrackselector::Beta,
+                       singletrackselector::TPCNClsFound,
+                       singletrackselector::TPCChi2NCl,
+                       singletrackselector::TPCCrossedRowsOverFindableCls,
+                       singletrackselector::TPCNClsShared,
+                       singletrackselector::ITSNCls,
+                       singletrackselector::ITSChi2NCl,
+                       singletrackselector::Sign,
+                       singletrackselector::Eta,
+                       singletrackselector::Phi,
+                       singletrackselector::StoredTOFNSigmaPr,
+                       singletrackselector::StoredTPCNSigmaPr,
+                       singletrackselector::StoredTOFNSigmaDe,
+                       singletrackselector::StoredTPCNSigmaDe,
+                       singletrackselector::Energy<singletrackselector::P>,
+                       singletrackselector::Pt<singletrackselector::P, singletrackselector::Eta>,
+                       singletrackselector::Px<singletrackselector::P, singletrackselector::Eta, singletrackselector::Phi>,
+                       singletrackselector::Py<singletrackselector::P, singletrackselector::Eta, singletrackselector::Phi>,
+                       singletrackselector::Pz<singletrackselector::P, singletrackselector::Eta>,
+                       singletrackselector::TOFNSigmaPr<singletrackselector::StoredTOFNSigmaPr>,
+                       singletrackselector::TPCNSigmaPr<singletrackselector::StoredTPCNSigmaPr>,
+                       singletrackselector::TOFNSigmaDe<singletrackselector::StoredTOFNSigmaDe>,
+                       singletrackselector::TPCNSigmaDe<singletrackselector::StoredTPCNSigmaDe>);
+
+} // namespace o2::aod
+#endif // PWGCF_DATAMODEL_SINGLETRACKSELECTOR_H_
+
+template <typename TrackType>
+inline bool TPCselection(TrackType const& track, std::pair<int, std::vector<float>> const& PIDcuts)
+{
```

Comment:
This structure contains a PDG/PIDcuts pair and if I'm not mistaken the pair is created in only ones at the beginning of a task (QA of PairTask) and this function only reads it since it's by a "const" reference.